src/tests/ampTicker.ts

```diff
@@ -0,0 +1,61 @@
+import fetch from 'node-fetch';
+
+interface TickerData {
+    total: number;
+    goal: number;
+}
+
+type TickerCountType = 'money' | 'people';
+
+export interface AMPTicker {
+    percentage: string;
+    goalAmountFigure: string;
+    goalAmountCaption: string;
+    currentAmountFigure: string;
+    currentAmountCaption: string;
+}
+
+const parse = (json: any): Promise<TickerData> => {
+    const total = parseInt(json.total, 10);
+    const goal = parseInt(json.goal, 10);
+
+    if (!Number.isNaN(total) && !Number.isNaN(goal)) {
+        return Promise.resolve({
+            total,
+            goal,
+        });
+    }
+    return Promise.reject(Error(`Failed to parse ticker data: ${json}`));
+};
```

Comment:
I guess this doesn't need to return a promise, but maybe this keeps it neater.

Reply:
This is pretty much just copied straight from the non-AMP ticker implementation on DCR. It's a shame having duplicated code on two codebases, but I feel it's worth keeping it consistent.

Reply:
👍